src/FilterWrapper/FilterWrapper.tsx

```diff
@@ -4,6 +4,7 @@ import * as React from "react";
 import { IconSvg } from "../IconSvg";
 import { FilterActionsPopover, InfoWrapper } from "./components";
 import { CallbackProps, SelectionStateType, StateProps } from "./types";
+import { neitralZoneClass } from "../HOCs";
```

Comment:
По-английски всё-таки `neutral`
```suggestion
import { neutralZoneClass } from "../HOCs";
```
И если там лежит название CSS-класса, то
```suggestion
import { neutralZoneClassName } from "../HOCs";
```